packages/calling/package.json

```diff
@@ -39,8 +39,8 @@
   },
   "dependencies": {
     "@types/platform": "1.3.4",
-    "@webex/internal-media-core": "2.0.0",
-    "@webex/media-helpers": "workspace:*",
+    "@webex/internal-media-core": "2.2.2",
+    "@webex/media-helpers": "workspace:^",
```

Comment:
Can we use `workspace:*` itself? It is important to keep it absolute so that there's no version conflicts when using meetings and messaging SDKs together

Reply:
take care of this change